interface_specifications/pe_openapi/apis/v1/c0_pe_definition_post.md

```diff
@@ -0,0 +1,56 @@
+### Presentation Definition POST
+
+`POST /pe/v1/{thid}/definition/`
```

Comment:
-> /definitions

In other places we have the thid in the JSON in the request body. Probably from a privacy perspective that is more desirable, as the thread interactions do not show up identifiable in access logs. At least we need to make sure that we choose to have thid either as a path param only, or as json body value. Currently I am leaning towards the latter

Reply:
fixed.